### ISSUE TITLE
setLaguage return undefined and doesn't switch language

### ISSUE THREAD

**Issue by fouad1994** [stefalda/ReactNativeLocalization#173]
`    this.languages = new LocalizedStrings({
            "en-US":{
                how:"How do you want your egg today?",
                boiledEgg:"Boiled egg",
                softBoiledEgg:"Soft-boiled egg",
                choice:"How to choose the egg"
            },
            en:{
                how:"How do you want your egg today?",
                boiledEgg:"Boiled egg",
                softBoiledEgg:"Soft-boiled egg",
                choice:"How to choose the egg"
            },
            ar: {
                how:"Come vuoi il tuo uovo oggi?",
                boiledEgg:"Uovo sodo",
                softBoiledEgg:"Uovo alla coque",
                choice:"Come scegliere l'uovo"
            }
        });`

this.languages.how //return "How do you want your egg today?"

but

languages.setLanguage('en') //return undefined

**Comment by stefalda** [comment]
That’s correct. SetLanguage has no return value. You should then use the string key to return the translation...

**Comment by HeroSony** [comment]
I think you need to re-render state:
```javascript
handleSetLanguage = key => {
    strings.setLanguage(key);
    this.setState({}); <--
};
```

**Comment by minhtet7674** [comment]
HeroSony please explain!
why use this statement ' this.setState({}); <-- ' ?

**Comment by HeroSony** [comment]
@minhtet7674 because this.setState will reload the whole component.
